pkg/worker/worker.go

```diff
@@ -128,15 +128,24 @@ func (vg *ValueGenerator) Generate() []byte {
 		// huge message sizes (e.g. 128MIB of zeros compresses down to <1MiB.
 		return compressible_payload
 	} else {
-		payload := make([]byte, vg.PayloadSize)
-		// An incompressible high entropy payload
-		n, err := rand.Read(payload)
+		randBytes := make([]byte, vg.PayloadSize)
+		// An incompressible high entropy payload. This will likely not be UTF-8 decodable.
+		n, err := rand.Read(randBytes)
 		if err != nil {
 			panic(err.Error())
 		}
 		if n != int(vg.PayloadSize) {
 			panic("Unexpected byte count from rand.Read")
 		}
+		// Convert to a valid UTF-8 string, replacing bad chars with " ".
+		payload := []byte(strings.ToValidUTF8(string(randBytes), " "))
```

Comment:
Please add a comment why we are making valid utf-8 bytes.

Reply:
Done